citi-onboarding/maracatronics
# hotfix(margin-who-we-are): a fix in margin of who we are section

## What i did
- Fix a problem in margin of "Quem somos" section

## How to test
1. First, you need to use the command ```git fetch --all```
2. Change your branch to "hotfix/margin-who-we-are" with ```git checkout hotfix/margin-who-we-are```
3. Now, you need to open two terminals
4. In some terminal, go to client folder, with ```cd client```
5. In the other terminal, go to server folder with ```cd server```
6. In each terminal, run it with ```npm/yarn start``` in the client folder and ```nodemon index.js``` in the server.
7. Now, you need to open the http://localhost:3000/ (the client side)
8. Lastly, you check if the distance between the elements in "Quem somos" section is alright and nice.
